docker/gpu/dockerfile.gpu

```diff
@@ -63,18 +63,18 @@ RUN mkdir -p /etc/OpenCL/vendors && \
 #           CONDA
 #################################################################################################################
 
-ARG CONDA_DIR=/opt/conda
+ARG CONDA_DIR=/opt/miniforge
 # add to path
 ENV PATH $CONDA_DIR/bin:$PATH
 
-# Install miniconda
+# Install miniforge
 RUN echo "export PATH=$CONDA_DIR/bin:"'$PATH' > /etc/profile.d/conda.sh && \
-    curl -sL https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -o ~/miniconda.sh && \
-    /bin/bash ~/miniconda.sh -b -p $CONDA_DIR && \
-    rm ~/miniconda.sh
+    curl -sL https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh -o ~/miniforge.sh && \
+    /bin/bash ~/miniforge.sh -b -p $CONDA_DIR && \
+    rm ~/miniforge.sh
 
 RUN conda config --set always_yes yes --set changeps1 no && \
-    conda create -y -q -n py3 python=3.8 mkl numpy scipy scikit-learn jupyter notebook ipython pandas matplotlib
+    conda create -y -q -n py3 numpy scipy scikit-learn jupyter notebook ipython pandas matplotlib
```

Comment:
Just use default Python to not update this place each time some Python version reaches EOL.